src/main/java/seedu/bigpp/exceptions/builderexceptions/BuilderMinBudgetException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.bigpp.exceptions.builderexceptions;
+
+import seedu.bigpp.exceptions.PPException;
+
+public class BuilderMinBudgetException extends PPException {
+    @Override
+    public String getMessage() {
+        return "You have set a budget that is lower than the current cost of the build, please set a higher budget.";
+    }
+}
```

Comment:
I think instead of not allowing the user to edit budget to lower, and not allowing the user to add items when above budget. We should just highlight to the user that it is overbudget, and instead make the build `!isComplete`

Reply:
this is for prebuilts where budget is infinite. when they make a copy, i dont want to allow them to set a budget lower than the current cost of the prebuilt pc